Dockerfile

```diff
@@ -40,6 +40,7 @@ ENV SC2PATH /home/app/mava/3rdparty/StarCraftII
 COPY . /home/app/mava
 RUN python -m pip uninstall -y enum34
 RUN python -m pip install --upgrade pip
+RUN pip install pyparsing==3.0.3
```

Comment:
Could we rather enforce this in `setup.py`, under `flatland-requirements`, instead of in the Dockerfile?
Seems a little neater to me.

Reply:
Fixed.

Reply:
Unfixed.